src/stream-transformer.ts

```diff
@@ -12,10 +12,10 @@ interface OpenAIChoice {
 
 interface OpenAIDelta {
 	role?: string;
-	content?: string;
+	content?: string | null;
 	reasoning?: string;
 	reasoning_content?: string | null;
-	tool_calls?: null;
+	tool_calls?: { index: number; id: string; type: "function"; function: { name: string; arguments: string } }[];
```

Comment:
Can you make this an interface aswell? Since this tool_calls type is quite big to be inlined